nntrainer/tensor/cl_operations/blas_kernels_fp16.cpp

```diff
@@ -291,7 +291,8 @@ void sgemm_cl(bool TransA, bool TransB, const __fp16 *A, const __fp16 *B,
   } while (false);
 }
 
-void addition_cl(const __fp16 *input, __fp16 *res, unsigned int size) {
+void addition_cl(const __fp16 *input, __fp16 *res, unsigned int size_input,
+                 unsigned int size_res) {
```

Comment:
What about updating `__fp16` into `_FP16` ?
Also, since header file is defining this function for fp16 only under the specific condition, it should be updated following the same policy.

```suggestion
#ifdef ENABLE_FP16
void addition_cl(const _FP16 *input, _FP16 *res, unsigned int size_input,
                 unsigned int size_res) {
```

Reply:
This file will only be compiled for fp16, hence I think the ifdef check is not required. Updated __fp16 into _FP16 though.

Reply:
Oh, I was confused it. It's my bad. Sorry for the confusion 🤣